### PR TITLE
Add support to Lustre backup

### PR DESCRIPTION
**Pull request by yiweng-amz**
**Is this a bug fix or adding new feature?**

**What is this PR about? / Why do we need it?**

**What testing is done?**

### CODE REVIEW

**Comment by Jeffwan** [comment]
I fetch your PR locally and test pass, i just rerun unit test to have a see. Since e2e pass as well, it might be flaky issue.

```
Ran 26 of 70 Specs in 0.094 seconds
SUCCESS! -- 26 Passed | 0 Failed | 0 Pending | 44 Skipped
--- PASS: TestSanity (0.09s)
PASS
ok  	github.com/kubernetes-sigs/aws-fsx-csi-driver/tests/sanity	2.078s
```

**Comment by Jeffwan** [comment]
One more thing. Can you squash your commits into one. Seem PR is not large and it's for one issue. One commit makes more sense, this repo use merge but not squash merge.

File: examples/kubernetes/dynamic_provisioning/specs/storageclass.yaml [hunk 1]
```diff
@@ -7,5 +7,8 @@ parameters:
   subnetId: subnet-0930cbd463b9688c5
   securityGroupIds: sg-03a4b6bd8afb19051,sg-05a37bfe01467059a
   deploymentType: PERSISTENT_1
+  automaticBackupRetentionDays: "1"
+  dailyAutomaticBackupStartTime: "00:00"
+  copyTagsToBackups: "true
```
Comment (wongma7) [inline examples/kubernetes/dynamic_provisioning/specs/storageclass.yaml:12]: extra quote/missing quote here

**Comment by wongma7** [comment]
/lgtm
/approve

There is a syntax error in the example though

/hold

**Comment by wongma7** [comment]
/hold cancel
/lgtm

tx

**Pull request merged**